quickstarts/103-demo-tenant/README.md

```diff
@@ -0,0 +1,35 @@
+# Demo Tenant Example (103 level)
```

Comment:
We generate docs. You need to crete .tmpl readme file like in other quickstarts

Reply:
Added [README.md.tmpl](https://github.com/Grant-Archibald-MS/power-platform-terraform-quickstarts/blob/grant-archibald-ms/demo-tenant/quickstarts/103-demo-tenant/README.md.tmpl) and removed README.md

Reply:
Sorry, forgot to tell you that you still have to run docs generation manually (we don't do that automatically in the repo at the moment)

https://github.com/Grant-Archibald-MS/power-platform-terraform-quickstarts/tree/grant-archibald-ms/demo-tenant/tools/quickstartgen 

Once everything is ok, you should see your quick start linked at the bottom of the main readme.md file

Reply:
@mawasile `go run quickstartgen.go` run and README.md generated